### ISSUE TITLE
metadata-generation-failed error

### ISSUE THREAD

**Issue by santosrai** [Nv7-GitHub/googlesearch#48]
I encountered an error while trying to install a package using pip. The error message is "metadata-generation-failed".

### Step to Reproduce
Run the following command in the terminal:
python3 -m pip install googlesearch-python

```py
Collecting googlesearch-python
  Using cached googlesearch-python-1.2.0.tar.gz (7.4 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1

note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

```
### Version used for development
pip 23.0.1
Python 3.10.5

**Comment by Nv7-GitHub** [comment]
Hmm, I just pushed this release. Perhaps there is an issue with the wheel

**Comment by Nv7-GitHub** [comment]
Oh, I believe I was missing a `setup.cfg` file, been a while since I published a version and apparently this is needed now! Was this the cause of the issue? Try installing version `1.2.1`

**Comment by CrazyWillBear** [comment]
I just tried installing 1.2.1, still not working

**Comment by Nv7-GitHub** [comment]
Oh sorry, it seems there was another issue with `requirements.txt` not being included, I pushed `1.2.2` to fix that, does that work?
